.gitignore

```diff
@@ -1,15 +1,16 @@
-#maven
-target/
-
-#eclipse
-.classpath
-.project
-.settings/
-
-#DS_Store
-.DS_Store
-
-#intellij
-.idea/
-*.iml
-*.iws
+#maven

```

Comment:
There are a number of changes in the line endings, it seems to be a Linux/Windows problem.
Can you please revert them?